website/docs/r/stream_analytics_output_servicebus_topic.html.markdown

```diff
@@ -73,6 +73,10 @@ The following arguments are supported:
 
 * `property_columns` - (Optional) A list of property columns to add to the Service Bus Topic output.
 
+* `system_property_columns` - (Optional) A key-value pair of system property columns for the Service Bus Topic Output.
+
+-> **NOTE:** The acceptable keys are `ContentType`, `CorrelationId`, `Label`, `MessageId`, `PartitionKey`, `ReplyTo`, `ReplyToSessionId`, `ScheduledEnqueueTimeUtc`, `SessionId`, `TimeToLive` and `To`.
```

Comment:
If these are the only column names that are accepted by the API then we can add these to the validation of the field instead of `ValidateFunc: validation.StringIsNotEmpty`.

Reply:
Actually, I already tried before, seems `pluginsdk.TypeMap` doesn't work with `ValidateFunc: validation.StringInSlice([]string{})`.

Below is code snippet that I tried:
```
"system_property_columns": {
				Type:     pluginsdk.TypeMap,
				Optional: true,
				Elem: &pluginsdk.Schema{
					Type: pluginsdk.TypeString,
					ValidateFunc: validation.StringInSlice([]string{
						"ContentType",
						"CorrelationId",
						"Label",
						"MessageId",
						"PartitionKey",
						"ReplyTo",
						"ReplyToSessionId",
						"ScheduledEnqueueTimeUtc",
						"SessionId",
						"TimeToLive",
						"To",
					}, false),
				},
			},
```